### ISSUE TITLE
Display Units in System Solve Results

### ISSUE THREAD

**Issue by VanHulleOne** [mgreminger/EngineeringPaper.xyz#199]
I think it would be nice to have the units shown in the System Solve results so I do not have to create another cell to confirm I haven't messed up my units somewhere.

Thanks again for this product and keep up the great work Mike!

**Comment by mgreminger** [comment]
Yes, the display of results in system solve cells is rather limited and can be confusing since the results are always in SI units even though no units are shown. This is closely related to #198. The system solve cells are basically showing the raw output of SymPy's solve function. It would require quite a bit of effort to display results in a system solve cell in the same way they are shown in a query statement. Because of this, the work around that you mentioned of adding a query statement to check results will be required for a while.
